packages/framework-core/src/sensor/health/health-indicators/booster-version.ts

```diff
@@ -0,0 +1,22 @@
+import * as path from 'path'
+import * as process from 'process'
+import { getLogger } from '@boostercloud/framework-common-helpers'
+import { BoosterConfig } from '@boostercloud/framework-types'
+
+export function boosterVersion(config: BoosterConfig) {
+  const projectAbsolutePath = path.resolve(process.cwd())
+  const logger = getLogger(config, 'boosterVersion')
+  try {
+    const packageJsonContents = require(path.join(projectAbsolutePath, 'package.json'))
+    const version = packageJsonContents.dependencies['@boostercloud/framework-core']
+    if (!version) {
+      logger.warn('Could not get Booster Version')
+      return ''
+    }
+    const versionParts = version.replace('workspace:', '').replace('^', '').replace('.tgz', '').split('-')
+    return versionParts[versionParts.length - 1]
```

Comment:
Shouldn't we calculate the Booster version based on the `package-lock.json`? We could have any semantic version in `package.json` file, but to know the exact version that is used, we need to look at the lock file

Reply:
Ok